utils_nlp/models/bert/sequence_classification.py

```diff
@@ -13,7 +13,7 @@
 from tqdm import tqdm
 
 from utils_nlp.models.bert.common import Language
```

Comment:
Do we still want to keep the models/bert folder? I thought it's going to be replaced by the transformer folder.

Reply:
I'll remove it if no one needs it - let's double check first